### ISSUE TITLE
Unable to navigate application using keyboard-only

### ISSUE THREAD

**Issue by ndierauf** [files-community/Files#14984]
### What feature or improvement do you think would benefit Files?

I am unable to navigate this application using keyboard-only. At a minimum, this app should provide all of the accessibility features that Windows File Explorer provides. In particular, the user should be able to select a file/folder using tab and arrow keys, then highlight the file/folder by tapping on the space-key, and then drill into the file/folder by hitting Enter key.

### Requirements

The user should be able to select a file/folder using tab and arrow keys, then highlight the file/folder by tapping on the space-key, and then drill into the file/folder by hitting Enter key.

### Files Version

Latest, as of March 14, 2024

### Windows Version

10

### Comments

I wanted so much for this app to be useful, but I am unable to use the app productively in its current state without being able to fully navigate using just a keyboard.

**Comment by Josh65-2201** [comment]
Thanks for the feedback, Those keys already work in Files and items are selected as you use the arrow keys. Please open a bug report with how you are trying to start with navigation or be more specific with what area or key aren't working.

**Comment by yaira2** [comment]
@ndierauf the you for the report, the keyboard shortcut for toggling file selection is `ctrl` + `space`. Please let us know if that's not working for you.

**Comment by ndierauf** [comment]
I just downloaded and installed Files again and was not able to reproduce the issue described above in the main files panel. I don't know if the issue was fixed in a subsequent app version, or if the app had gotten into a corrupt state last week that prevented me from opening files from the keyboard.
One thing that I am noticing is that selecting folders in the left-panel, ie folders in Favorites, sometimes I can drill down into a selected folder and sometimes I cannot, which is an inconsistent behavior.
I don't have time to provide thorough QA for the app. I wish you success moving forward. Feel free to close this ticket out.

**Comment by yaira2** [comment]
> One thing that I am noticing is that selecting folders in the left-panel, ie folders in Favorites, sometimes I can drill down into a selected folder and sometimes I cannot, which is an inconsistent behavior.

A tree view isn't supported yet but it's on the todo list. In the meantime you can try the Column View which is an alternative method of drilling into folders.
